[2025-01-26T22:18:07.895+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-26T22:18:07.947+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_processing.nds-load scheduled__2025-01-26T22:16:00+00:00 [queued]>
[2025-01-26T22:18:07.967+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_processing.nds-load scheduled__2025-01-26T22:16:00+00:00 [queued]>
[2025-01-26T22:18:07.967+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-26T22:18:08.004+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): nds-load> on 2025-01-26 22:16:00+00:00
[2025-01-26T22:18:08.022+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2022) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-26T22:18:08.025+0000] {standard_task_runner.py:72} INFO - Started process 2027 to run task
[2025-01-26T22:18:08.027+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_processing', 'nds-load', 'scheduled__2025-01-26T22:16:00+00:00', '--job-id', '476', '--raw', '--subdir', 'DAGS_FOLDER/DAG_etl.py', '--cfg-path', '/tmp/tmpar6e3ojm']
[2025-01-26T22:18:08.034+0000] {standard_task_runner.py:105} INFO - Job 476: Subtask nds-load
[2025-01-26T22:18:08.167+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_processing.nds-load scheduled__2025-01-26T22:16:00+00:00 [running]> on host a664bcf281bf
[2025-01-26T22:18:08.934+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_processing' AIRFLOW_CTX_TASK_ID='nds-load' AIRFLOW_CTX_EXECUTION_DATE='2025-01-26T22:16:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-26T22:16:00+00:00'
[2025-01-26T22:18:08.938+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-26T22:18:09.017+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:18:09.104+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:18:09.227+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:18:09.388+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:18:09.510+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:18:09.660+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:18:09.783+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:18:09.926+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:18:10.114+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:18:10.245+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-26T22:18:10.291+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-26T22:18:10.292+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_processing, task_id=nds-load, run_id=scheduled__2025-01-26T22:16:00+00:00, execution_date=20250126T221600, start_date=20250126T221807, end_date=20250126T221810
[2025-01-26T22:18:10.396+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-26T22:18:10.485+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-26T22:18:10.488+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
